packages/playwright-core/src/utils/happy-eyeballs.ts

```diff
@@ -132,9 +134,13 @@ export async function createConnectionAsync(
         port: options.port as number,
         host: address });
 
+    (socket as any).dnsLookupAt = dnsLookupAt;
```

Comment:
I think we should make this a proper part of the API in some way.

Reply:
Agree it's a little hacky, but I can't find anything better. How would you do it?

Reply:
Create a function here called `timingForSocket(socket)` that can read values that were set through symbols to avoid clashing with proper fields. Basically, encapsulate the internal symbols to this file.